src/bookmark.h

```diff
@@ -24,6 +24,7 @@ gboolean bookmark_add(const char *uri, const char *title, const char *tags);
 gboolean bookmark_remove(const char *uri);
 gboolean bookmark_fill_completion(GtkListStore *store, const char *input);
 gboolean bookmark_fill_tag_completion(GtkListStore *store, const char *input);
+gboolean build_html_file();
```

Comment:
I think you should keep `bookmark_` prefix for exported function. It would be more evident where the function lives, and what it should do.